### PR TITLE
create order

### PR DESCRIPTION
**Pull request by lolesia**
- Order models
- change views.py 
- change urls

### CODE REVIEW

File: Backend/test_project/order/models.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    @property
+    def total_price(self):
+        total = 0
+        for product in self.name.all():
+            total += product.price * self.quantity
+        return total
+
+    class Meta:
+        ordering = ['data']
```
Comment (Kozatskiy-Artem) [inline Backend/test_project/order/models.py]: Data to date

File: Backend/test_project/order/models.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    quantity = models.IntegerField(default=1, verbose_name='Кількість')
+    user = models.ForeignKey(User, on_delete=models.SET('видалений користувач'), null=True ,verbose_name='Користувач')
+    first_name = models.CharField(max_length=30, verbose_name="Ім'я")
+    last_name = models.CharField(max_length=50, verbose_name='Прізвище')
+    phone = models.CharField(max_length=30, verbose_name='Номер телефону')
+    email = models.CharField(max_length=100, null=True, blank=True)
+    сity = models.CharField(max_length=30, verbose_name='Ваше місто')
+    delivery_way = models.CharField(max_length=10, choices=DELIVERY, verbose_name='Спосіб доставки')
+    payment = models.CharField(max_length=10, choices=PAYMENT)
+    data = models.DateField(auto_now=True)
```
Comment (Kozatskiy-Artem) [inline Backend/test_project/order/models.py]: Change data to date

File: Backend/test_project/order/models.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    @property
+    def total_price(self):
+        total = 0
+        for product in self.name.all():
```
Comment (Kozatskiy-Artem) [inline Backend/test_project/order/models.py]: Change name to products

File: Backend/test_project/order/serializers.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+from .models import Order
+from users.models import User
+from product.models import Product
+
+
+class OrderSerializer(serializers.ModelSerializer):
+    user = serializers.SlugRelatedField(slug_field='user', queryset=User.objects.all())
+    name = serializers.SlugRelatedField(many=True, slug_field='name', queryset=Product.objects.all())
```
Comment (Kozatskiy-Artem) [inline Backend/test_project/order/serializers.py]: Name to products

**Review by antony-kosenko — APPROVED** [review]

**Review by Kozatskiy-Artem — APPROVED** [review]

**Pull request merged**